web/satellite/src/views/register/Register.vue

```diff
@@ -46,7 +46,15 @@
                 this.secret = this.$route.query.token.toString();
             }
 
-            let { ids } = this.$route.params;
+            let { ids = '' } = this.$route.params;
+            let decoded = '';
+            try {
+                decoded = atob(ids);
+            } catch {
+                this.$router.push(ROUTES.LOGIN);
```

Comment:
would it  be better to notify user about invalid id, instead of redirecting him on login page?
redirect without information message could confuse

Reply:
Good point! I changed it to be a error message